spec/plsql/schema_spec.rb

```diff
@@ -86,7 +86,7 @@
   it "should not connect with wrong port number" do
     expect {
       plsql.connect! @username, @password, :host => @host, :port => 9999, :database => @database
-    }.to raise_error(/no listener|could not establish the connection/)
+    }.to raise_error(/ORA-12541/)
```

Comment:
In fact, now that you mention it, it think it would be better not to set the ENV parameters at  all, or set them to some defaults, if they are not set.
That should go for:
NLS_LANG Oracle session language
ORA_SDTZ Oracle session timezone 
TZ ruby timezone parameter
So by default you would get:
```
TZ=UTC
ORA_SDTZ=UTC
NLS_LANG=AMERICAN_AMERICA.AL32UTF8
```
Anyway, users should be able to change those settings as they need, to setup any valid language and timezone on their environment.
I can update the `spec_helper.rb` to reflect that.
Would that be OK?